cmd/vault-subpath-proxy/main.go

```diff
@@ -56,7 +63,23 @@ func main() {
 		logrus.WithError(err).Fatal("Failed to get opts")
 	}
 
-	server, err := createProxyServer(opts.vaultAddr, opts.listenAddr, opts.kvMountPath)
+	var kubeconfigs map[string]*rest.Config
+	if os.Getenv("KUBECONFIG") != "" || opts.kubeconfig != "" {
+		kubeconfigs, _, err = util.LoadKubeConfigs(opts.kubeconfig, nil)
```

Comment:
Why does this one not need a restart-on-kubeconfig-change func?

Reply:
Added code to reload clients if the kubeconfig changes